### PR TITLE
Add my custom layouts for GH60, DZ60 and Minivan

### PR DESCRIPTION
**Pull request by danbee**
## Description

I use QMK for my GH60, DZ60 and Minivan keyboards. These are the keymap
files I have put together for those keyboards/PCB's.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by danbee** [comment]
Hi @fauxpark, thanks for your feedback!

I've fixed up everything you commented on. Please let me know if there's anything I missed.

File: keyboards/dz60/keymaps/danbee/keymap.c [hunk 1]
```diff
@@ -0,0 +1,66 @@
+#include QMK_KEYBOARD_H
+
+// Name layers
+#define _QW 0
+#define _F1 1
+#define _F2 2
```
Comment (noroadsleft) [inline keyboards/dz60/keymaps/danbee/keymap.c]: Completely optional, but using an `enum` here makes this section easier to maintain.

```c
enum layer_names {
    _QW,
    _F1,
    _F2,
};
```

This is equivalent to what you have now once compiled.

File: keyboards/gh60/revc/keymaps/danbee/keymap.c [hunk 2]
```diff
@@ -0,0 +1,66 @@
+#include QMK_KEYBOARD_H
+
+// Name layers
+#define _QW 0
+#define _F1 1
+#define _F2 2
```
Comment (noroadsleft) [inline keyboards/gh60/revc/keymaps/danbee/keymap.c]: Same here as above.

File: keyboards/thevankeyboards/minivan/keymaps/danbee/keymap.c [hunk 3]
```diff
@@ -0,0 +1,59 @@
+#include QMK_KEYBOARD_H
+
+#define _QW 0
+#define _L1 1
+#define _L2 2
```
Comment (noroadsleft) [inline keyboards/thevankeyboards/minivan/keymaps/danbee/keymap.c]: And here.

File: keyboards/dz60/keymaps/danbee/keymap.c (hunk 1 above)
Comment (danbee) [reply keyboards/dz60/keymaps/danbee/keymap.c]: This is a good suggestion, thanks!

**Comment by drashna** [comment]
Though, we really do recommend it.  If you want to leave it, that's fine. But let us know, that way we can merge it if you don't want to change it.

**Comment by danbee** [comment]
@drashna @noroadsleft @fauxpark Thanks for all your feedback! I'm pretty sure this is now ready to merge.

File: keyboards/thevankeyboards/minivan/keymaps/danbee/keymap.c [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    │ § │Br-│Br+│   │   │   │   │   │   │ - │ = │ Eject│
+    ├───┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬─────┤
+    │    │   │   │   │   │   │Hom│PgD│PgU│End│Ins│     │
+    ├────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬───┤
+    │      │   │   │   │   │Prv│Nxt│Mut│Vo-│Vo+│   │   │
+    ├────┬─┴───┼───┴┬──┴───┴─┬─┴───┴─┬─┴──┬┴───┴┬──┴───┤
+    │    │     │    │        │ Pause │    │     │      │
+    └────┴─────┴────┴────────┴───────┴────┴─────┴──────┘*/
+  [_L2] = LAYOUT( /* LAYER 2 */
+    KC_NUBS, KC_BRMD, KC_BRMU, _______, RESET,   _______, _______, _______, _______, KC_MINS, KC_EQL,  KC_EJCT,
```
Comment (fauxpark) [inline keyboards/thevankeyboards/minivan/keymaps/danbee/keymap.c]:
```suggestion
    KC_NUBS, KC_BRID, KC_BRIU, _______, RESET,   _______, _______, _______, _______, KC_MINS, KC_EQL,  KC_EJCT,
```

File: keyboards/thevankeyboards/minivan/keymaps/danbee/keymap.c [hunk 5]
```diff
@@ -0,0 +1,62 @@
+#include QMK_KEYBOARD_H
+
+enum layer_names {
+  _QW,
+  _L1,
+  _L2,
+};
+
+#define CTL_TAB LCTL_T(KC_TAB)
+#define L2_QUOT LT(2, KC_QUOT)
```
Comment (fauxpark) [inline keyboards/thevankeyboards/minivan/keymaps/danbee/keymap.c]:
```suggestion
#define L2_QUOT LT(_L2, KC_QUOT)
```

File: keyboards/dz60/keymaps/danbee/keymap.c [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+    │     │   │   │   │   │   │   │   │   │   │   │   │   │     │
+    ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐    │
+    │ Caps │   │   │   │   │   │Hom│PgD│PgU│End│   │   │   │    │
+    ├────┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┴────┤
+    │    │   │   │   │   │   │Rst│   │   │   │   │   │          │
+    ├────┼───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬────┬────┤
+    │    │    │    │                        │    │    │    │    │
+    └────┴────┴────┴────────────────────────┴────┴────┴────┴────┘*/
+  [_L2] = LAYOUT_60_iso( /* Layer 2: Functions */
+    KC_POWER, KC_F1,  KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_EJCT,
```
Comment (fauxpark) [inline keyboards/dz60/keymaps/danbee/keymap.c]:
```suggestion
    KC_PWR,   KC_F1,  KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_EJCT,
```

File: keyboards/gh60/revc/keymaps/danbee/keymap.c [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+    │     │   │   │   │   │   │   │   │   │   │   │   │   │     │
+    ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐    │
+    │ Caps │   │   │   │   │   │Hom│PgD│PgU│End│   │   │   │    │
+    ├────┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┴────┤
+    │    │   │   │   │   │   │   │   │   │   │   │   │          │
+    ├────┼───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬────┬────┤
+    │    │    │    │                        │    │    │    │    │
+    └────┴────┴────┴────────────────────────┴────┴────┴────┴────┘*/
+  [_L2] = LAYOUT_60_iso( /* Layer 2: Functions */
+    KC_POWER, KC_F1,  KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_EJCT,
```
Comment (fauxpark) [inline keyboards/gh60/revc/keymaps/danbee/keymap.c]:
```suggestion
    KC_PWR,   KC_F1,  KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_EJCT,
```

**Comment by danbee** [comment]
Fixed up and squashed!

**Comment by fauxpark** [comment]
Thanks!

**Pull request merged**